packages/terra-date-picker/src/DateInput.jsx

```diff
@@ -188,42 +189,50 @@ class DatePickerInput extends React.Component {
     const theme = this.context;
 
     const label = this.props.ariaLabel ? this.props.ariaLabel : intl.formatMessage({ id: 'Terra.datePicker.date' });
+    this.formatDescriptionId = (help === undefined) ? `terra-date-picker-description-format-${uuidv4()}` : '';
```

Comment:
Should `this.formatDescriptionId` just be set once in the constructor instead of generating a new id on every render?